src/frontend/src/expr/table_function.rs

```diff
@@ -57,6 +59,7 @@ impl TableFunctionType {
             TableFunctionType::Generate => "generate_series",
             TableFunctionType::Unnest => "unnest",
             TableFunctionType::RegexpMatches => "regexp_matches",
+            TableFunctionType::RawTable => "__rw_table",
```

Comment:
Suggest naming the type as `InternalTable`. BTW, the `rw` means RisingWave instead of "raw". 😄